src/cmds/restic/cmd_forget.go

```diff
@@ -55,8 +55,9 @@ func init() {
 	f.IntVarP(&forgetOptions.Yearly, "keep-yearly", "y", 0, "keep the last `n` yearly snapshots")
 
 	f.StringSliceVar(&forgetOptions.KeepTags, "keep-tag", []string{}, "always keep snapshots with this `tag` (can be specified multiple times)")
-	f.StringVar(&forgetOptions.Hostname, "hostname", "", "only forget snapshots for the given hostname")
-	f.StringSliceVar(&forgetOptions.Tags, "tag", []string{}, "only forget snapshots with the `tag` (can be specified multiple times)")
+	f.StringVar(&forgetOptions.Host, "host", "", "only consider snapshots for this `host`")
```

Comment:
Same comment as in the other PR: we're changing from `--hostname` to `--host`, we need to find a workaround for people using `--hostname` in their scripts.